src/Integration/Binding/SolutionBindingOperation.cs

```diff
@@ -278,8 +278,7 @@ private void PendBindingInformation(ConnectionInformation connInfo)
                 credentials, connInfo.Organization);
             bound.Profiles = map;
 
-            var config = new BindingConfiguration(bound, this.bindingMode);
```

Comment:
I'm working on passing the config location as a ctor param in BindingConfiguration, and this bit was in the way.
We don't even need to create BindingConfiguration, as the inner classes save the SonarQubeProject, and they are the ones that know the path in the first place.